src/app/(proper_react)/redesign/(public)/LandingView.test.tsx

```diff
@@ -38,14 +43,50 @@ describe("When Premium is not available", () => {
     );
     await user.type(inputField, "mail@example.com");
 
-    const submitButton = screen.getByRole("button", { name: "Get free scan" });
-    await user.click(submitButton);
+    const submitButton = screen.getAllByRole("button", {
+      name: "Get free scan",
+    });
+    await user.click(submitButton[0]);
 
     expect(signIn).toHaveBeenCalledTimes(1);
     expect(signIn).toHaveBeenCalledWith("fxa", expect.any(Object), {
       email: "mail@example.com",
     });
   });
+
+  it("shows the scanning for exposures illustration in the fix your exposures section", () => {
+    const ComposedDashboard = composeStory(LandingNonUs, Meta);
+    render(<ComposedDashboard />);
+
+    const scanningForExposuresIllustration = screen.getByTestId(
+      "scanning-for-exposures-image",
+    );
```

Comment:
As mentioned on Slack, [this](https://testing-library.com/docs/queries/about/#priority) is order of preference when it comes to selectors, with the ones closest to how a user interacts with the page being most preferred. Users can't see test IDs, which is why that option is listed last :)

So this matcher could have been:

```ts
    const scanningForExposuresIllustration = screen.getByRole(
      "img",
      { name: "Scanning for exposures progress bar" },
    );
```

And then the test would have caught a bug: both images have the same alt text :)

But as I suggested elsewhere, these images probably don't need alt texts. Which means that the user probably doesn't really interact with the image, which means that... We're back to finding it through a test ID :sweat_smile: